src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -336,12 +336,12 @@ private void Initialize(XmlReader reader, string fileName, bool ignoreErrors)
                     }
                     else
                     {
-                        InternalLogger.Error("Error in Parsing Configuration File. Exception : {0}", ConfigException);
+                        InternalLogger.Error(ConfigException, "Error in Parsing Configuration File.");
```

Comment:
The condition `if (LogManager.ThrowExceptions)` above can be removed. It will always be `false`.